README.md

```diff
@@ -28,6 +28,29 @@ pnpm add @api3/commons
 
 Read the documentation of each module how to use it in the project.
 
+### Import paths
+
+We want the consumers to import common utilities like this:
+
+```ts
+import { createLogger } from '@api3/commons/logger';
+// and not like this
+import { createLogger } from '@api3/commons/dist/logger';
+```
+
+This is done via `package.json` field called [exports](https://nodejs.org/api/packages.html#package-entry-points). This
+field works for both CJS and ESM starting from Node version 12. In order for TS to support this, you need to make sure
+you use `moduleResolution` and `module` set to `Node16`:
+
+```json
+{
+  "compilerOptions": {
+    "module": "Node16",
```

Comment:
Without that you would need to fallback to importing directly from the `dist` like in https://github.com/api3dao/signed-api/blob/5b10549bfc030bb8676b6509e5976e8f3ffe3164/packages/pusher/test/fixtures.ts#L2 (which works, but is ugly).

Considering that node16 is [past EoL](https://endoflife.date/nodejs) I don't think this a huge deal.

Reply:
> Considering that node16 is past EoL I don't think this a huge deal.

I agree, but I now to figure out why I have to put "Node16" in my tsconfig as someone wanting to use this 😅 (I don't know what `"module": "Node16"` means or the consequences of putting it there)

Reply:
I see you concern. What about rewriting it like this:
1. Describe that you need to import from `dist` when using `"commonjs"` as module resolution.
2. Describe the `tsconfig.json` changes to avoid imports from dist (recommended).

This way, you are not forced to do any `tsconfig` changes, but everything still gets documented.

Reply:
Yeah I like that 👍🏻